### PR TITLE
Integrate janus::server with needrestart::service

### PR DESCRIPTION
**Pull request by kris-lab**
Part of https://github.com/cargomedia/puppet-cargomedia/issues/1299
Depends on https://github.com/cargomedia/debian-packages/issues/132

### CODE REVIEW

**Comment by kris-lab** [comment]
retest this please

**Comment by njam** [comment]
How about we do it only for jessie? I understand for now it's needed for janus and cm-janus, which will be fully on jessie soon.

**Comment by kris-lab** [comment]
How would you do it then? We need still compatibility with Wheezy... or you mean to apply `needrestart::service{}` only for `jessie` and ignore for `wheezy`?

**Comment by njam** [comment]
Hm, yes why not an `if`. Once we're done with wheezy we can then grep for these IFs and delete them…

**Comment by kris-lab** [comment]
@njam please review

Recently the `janus-gateway-rtpbroadcast@0.0.22` has been released. It is not applied to production so I will test current logic with the manual upgrade.

File: modules/janus/manifests/server.pp [hunk 1]
```diff
@@ -110,6 +110,11 @@
     ],
   }
 
+  if $::facts['lsbdistcodename'] == 'jessie' {
```
Comment (njam) [inline modules/janus/manifests/server.pp]: I would suggest to do the opposize (`!= 'wheezy'`?), will be easier to search and remove…

Comment (kris-lab) [inline modules/janus/manifests/server.pp]: OK, however we will search by e.g. `lsbdistcodename` to be on the safe side.

**Comment by njam** [comment]
true, lgtm

Did you test all of it together?

**Comment by kris-lab** [comment]
I did some extended tests in my local VM/jessie. Seems to work perfectly.

Let's see how it goes on the production.

**Pull request merged**